bank/Accounts/models.py

```diff
@@ -0,0 +1,11 @@
+from django.db import models
+
+# Create your models here.
+class Users(models.Model):
+    first_name = models.CharField(max_length=50)
+    second_name = models.CharField(max_length=50)
+    account_name = models.CharField(max_length=100)
+    account_number = models.IntegerField(primary_key=True , unique=True)
```

Comment:
It would be best to maintain this as unique but not primary key

Reply:
Given such restrictions in IntegerField, https://docs.djangoproject.com/en/2.2/ref/models/fields/#integerfield, I think it would be best not to use it for account numbers that could run over a wider range of numbers.